### ISSUE TITLE
Inkscape on osx 10.13.x fails on goo/GooHash.h

### ISSUE THREAD

**Issue by rolandvs** [caskformula/homebrew-caskformula#55]
This story ended with an error when compiling:

/tmp/inkscape-20180524-74133-1kr9xyk/src/extension/internal/pdfinput/pdf-parser.cpp:41:10: fatal error: 'goo/GooHash.h' file not found
**#include "goo/GooHash.h"**
         ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [src/CMakeFiles/inkscape_base.dir/extension/internal/pdfinput/pdf-parser.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/inkscape_base.dir/all] Error 2
make: *** [all] Error 2

How to fix?

**Comment by Manikgarg** [comment]
Yes, I am getting the same error while installing on MacOS 10.13.4 using Homebrew formula:

/tmp/inkscape-20180524-11274-obpgpe/inkscape-0.92.2/src/extension/internal/pdfinput/pdf-parser.cpp:41:10: fatal error: 'goo/GooHash.h' file not found
#include "goo/GooHash.h"
         ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [src/CMakeFiles/inkscape_base.dir/extension/internal/pdfinput/pdf-parser.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [src/CMakeFiles/inkscape_base.dir/all] Error 2
make: *** [all] Error 2

**Comment by IgorFobia** [comment]
Same error, still for MacOS 10.13.4 
```
/tmp/inkscape-20180524-9882-pf13rn/inkscape-0.92.2/src/extension/internal/pdfinput/pdf-parser.cpp:41:10: fatal error: 'goo/GooHash.h' file not found
#include "goo/GooHash.h"
         ^~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [src/CMakeFiles/inkscape_base.dir/extension/internal/pdfinput/pdf-parser.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
/tmp/inkscape-20180524-9882-pf13rn/inkscape-0.92.2/src/extension/internal/pdfinput/svg-builder.cpp:1023:48: error: assigning to 'char *' from incompatible type 'const char *'
        _font_specification = font->getName()->getCString();
                              ~~~~~~~~~~~~~~~~~^~~~~~~~~~~~
1 error generated.
make[2]: *** [src/CMakeFiles/inkscape_base.dir/extension/internal/pdfinput/svg-builder.cpp.o] Error 1
make[1]: *** [src/CMakeFiles/inkscape_base.dir/all] Error 2
make: *** [all] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/caskformula/homebrew-caskformula/issues
```

**Comment by devonwaldon** [comment]
I wonder if it's related to https://bugs.launchpad.net/inkscape/+bug/1772883? Looks like pretty recent action on it from Inkscape's end.

**Comment by orestesgaolin** [comment]
It seems that the team patched the issue but I still cannot build Inkscape. How can we select the master branch as a source?

**Comment by weikengchen** [comment]
See https://github.com/caskformula/homebrew-caskformula/issues/54

**Comment by weikengchen** [comment]
maybe I should upgrade to 0.92.4...
